src/meta/test/CreateBackupProcessorTest.cpp

```diff
@@ -0,0 +1,149 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include <gtest/gtest.h>
+#include "common/base/Base.h"
+#include "common/fs/TempDir.h"
+#include "meta/processors/admin/CreateBackupProcessor.h"
+#include "meta/test/TestUtils.h"
+#include "utils/Utils.h"
+
+namespace nebula {
+namespace meta {
+
+#define RETURN_OK(req)                                                                             \
+    UNUSED(req);                                                                                   \
```

Comment:
Why is the parameter specified but not used?

Reply:
Because there is no need to parse req here for testing purposes.